app/configs/fuzz_features.conf

```diff
@@ -0,0 +1,48 @@
+# The goal of this file is to:
+# 1. Fuzz more code
+# 2. Reduce the gap between fuzzed SOF and the real thing.
+
+# KConfig warnings are NOT fatal so you must always INSPECT build logs when changing
+# .conf files. See https://github.com/thesofproject/sof/issues/9386
+
+# Note 1. is not as simple as enabling as many CONFIG_ as possible. Enabling some CONFIG_
+# can technically _disable_ some code paths. But the opposite is more common so let's add
+# more.
+
+# In the longer term we should have some more elaborate configuration framework to reduce
+# duplicate/diverge between fuzzing and production but also across product
+# generations. Something like config fragments including each other? Not something as
+# complicated as Yocto fragments but something more flexible than
+# https://docs.zephyrproject.org/latest/build/kconfig/setting.html#initial-conf
+#
+# Discuss in https://github.com/thesofproject/sof/issues/9386
+
+# Many of these features are too far from IPC to make any fuzzing difference. But as long
+# as they reduce the size of the textual difference between build-fuzz/zephyr/.config and
+# build-xxx/zephyr/.config, they make that manual comparison easier which is still a win.
+
+
+CONFIG_COUNTER=y
+
+CONFIG_PROBE=y
+
+CONFIG_CRYPTO=y
+
+CONFIG_LOG_TIMESTAMP_64BIT=y
+CONFIG_MM_DRV=y
+
+CONFIG_DMA=y
+
+CONFIG_DAI=y
+
+CONFIG_PM_DEVICE=y
+CONFIG_POWER_DOMAIN=y
```

Comment:
do we actually want / need to fuzz-test Zephyr Kconfig options or only SOF?

Reply:
There's a long comment at the top of this file... too long? :-)
Does line 20 answer the question?

Reply:
I think I was confusing things a bit. For some reason I decided that for fuzzing tests we also randomise .config, which isn't the case, is it? Our fuzzing .config is fully deterministic so it can be fixed as close to the production one as only possible?

Reply:
Yes that's the idea.

Even if we wanted to do it somewhere, `kconfiglib.py` has unfortunately no "randconfig" ability. At least not the last time I checked.